NextcloudTalk/NCButton.swift

```diff
@@ -26,18 +29,17 @@ struct NCButtonSwiftUI: View {
                 Text(title)
                     .bold()
                     .foregroundColor(titleColorForStyle(style: style).opacity(disabled ? 0.5 : 1))
-                    .padding(.vertical, 4)
-                    .padding(.horizontal, 40)
-                    .frame(height: height)
+                    .fixedSize(horizontal: false, vertical: true)
+                    .padding(.vertical, NCButtonVerticalPadding)
+                    .padding(.horizontal, NCButtonHorizontalPadding)
                     .background(backgroundColorForStyle(style: style).opacity(disabled ? 0.5 : 1))
-                    .cornerRadius(height / 2)
+                    .cornerRadius(NCButtonCornerRadius)
```

Comment:
To be more in line with the design refresh in NC30 ?

Reply:
Yes :) This PR is just for NCButton in SwiftUI.
The changes for non SwiftUI NCButtons will come in a follow up PR.